content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -100,7 +96,7 @@ The worker synchronizes ServiceNow with the current run state of Terraform works
 
 -> **Roles Required:** `admin` or `x_terraform.config_user`
 
-1. Exit Service Now Studio and return to the ServiceNow Service Management Screen.
+1. From the ServiceNow Service Management Screen.
 1. Using the left-hand navigation, open the configuration table for the integration to manage the Terraform Enterprise connection.
```

Comment:
Should these be indented? I assume they're actions you take from the service now management screen?

Reply:
Also, I'm not sure if we're officially lifting the restriction with using SNow with Terraform Cloud, but from convos with PMs it seemed like they were open to it. Probably out of scope for this PR but we might want to consider changing references to TFE to Cloud instead.